docker/config.yaml.mako

```diff
@@ -74,15 +74,17 @@ vars:
       use_wms: "true"
       validate: "false"
       # The following parameters are not used by xml2pdf, but might be in the fure (see issue #873)
-      buffer: 30
+      buffer: 10
       basic_map_size: [493, 280]
       pdf_dpi: 300
       pdf_map_size_millimeters: [174, 99]
   % else:
       # Configuration for MapFish-Print print service
       renderer: pyramid_oereb.contrib.print_proxy.mapfish_print.Renderer
-      # The minimum buffer in pixel at 72 DPI between the real estate and the map's border.
-      buffer: 30
+      # The minimum buffer in pixel at 72 DPI between the real estate and the map's border. If your print
+      # system draw a margin around the feature (the real estate), you have to set your buffer
+      # here accordingly. MapFishPrint set per default a (Map -> ZoomToFeatures ->) MinMargin to 10px.
```

Comment:
set -> sets

But I think we should not put this here, but instead add an explicit configuration line in the pyramid_oereb_mfp.

Reply:
Ok. I'll move this part in pyramid_oereb_mfp